### PR TITLE
Add overall FIPS status to ykman info

### PR DESCRIPTION
**Pull request by emlun**
This would merge into #123.

Does the output format look reasonable?

### CODE REVIEW

File: ykman/cli/info.py [hunk 1]
```diff
@@ -27,10 +27,29 @@
...
+from ..descriptor import open_device
+from ..fido import FipsU2fController
+from ..oath import OathController
+from ..otp import OtpController
+from ..util import APPLICATION, TRANSPORT
 import click
 
 
+def get_overall_fips_status(serial):
+    stati = {}
```
Comment (dagheyman) [inline ykman/cli/info.py]: `stati` ?

File: ykman/cli/info.py [hunk 2]
```diff
@@ -64,6 +80,23 @@ def info(ctx):
...
+
+        fips_status = get_overall_fips_status(dev.serial)
+
+        click.echo('FIPS mode active: {}'.format(
+            'Yes' if all(fips_status.values()) else 'No'))
+
+        status_keys = list(fips_status.keys())
+        status_keys.sort()
+        for status_key in status_keys:
+            click.echo('  - {}: {}'.format(
```
Comment (dagheyman) [inline ykman/cli/info.py]: I don't like the `-` (we dont use them in other places), let's just use tabs for indentation.

File: ykman/cli/info.py [hunk 3]
```diff
@@ -64,6 +80,23 @@ def info(ctx):
...
+                status_key, 'Yes' if fips_status[status_key] else 'No'))
+
+    click.echo()
```
Comment (dagheyman) [inline ykman/cli/info.py]: This one is just need if fips, leads to extra blank line on regular devices.

File: ykman/cli/info.py (hunk 1 above)
Comment (emlun) [reply ykman/cli/info.py]: Meh, seems like [my latin fu failed me](https://english.stackexchange.com/questions/877/what-is-the-plural-form-of-status).

File: ykman/cli/info.py (hunk 3 above)
Comment (emlun) [reply ykman/cli/info.py]: Good catch, thanks!

File: ykman/cli/info.py (hunk 2 above)
Comment (emlun) [reply ykman/cli/info.py]: Fixed!

**Comment by emlun** [comment]
Travis failure is due to a flake8 issue which is fixed in 6799a1753c1d1b49ecf60d0a9ab3dd3f50479ecd in #123; merging.

**Pull request merged**